R/write_dataset_json.R

```diff
@@ -65,18 +80,22 @@ write_dataset_json <- function(x, file, pretty=FALSE) {
         }
         x[y$name] <- strftime(as.numeric(x[[y$name]]), "%H:%M:%S", tz='UTC')
       }
+    } else if (float_as_decimals && y$dataType == "float") {
```

Comment:
Why only dataType == "float"?  what if the metadata has double our decimal?

Reply:
Makes sense and I'll add those.

Reply:
Updated